components/brave_rewards/browser/database/database_media_publisher_info.cc

```diff
@@ -0,0 +1,140 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <string>
+#include <utility>
+
+#include "base/bind.h"
+#include "base/strings/stringprintf.h"
+#include "brave/components/brave_rewards/browser/database/database_media_publisher_info.h"
+#include "sql/statement.h"
+#include "sql/transaction.h"
+
+namespace brave_rewards {
+
+namespace {
+  const char* table_name_ = "media_publisher_info";
```

Comment:
`table_name_` is not a member variable so no need for underscore at end of the name

Reply:
nit: `const char table_name[] = "media_publisher_info";`

Reply:
https://github.com/brave/brave-browser/issues/7145